profile.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Profile</title>
+  </head>
+  <body>
+    <header>
+      <h1>PROFILE</h1>
+    </header>
+
+    <section class="about">
+      <img src="xxx.png" alt="Profile picture" />
+      <p>Ricardo Bachmann</p>
+
+      <h3>What's up?!:</h3>
+      <p>My Name is...</p>
+
+      <lable for="about--tags">Add tags about yourself divided by comma:</lable>
```

Comment:
Your for-attribute needs to match the id of your textarea :)
```suggestion
      <label for="text--field__tags">Add tags about yourself divided by comma:</label>
```

Reply:
makes sense :D